api/src/api/v1/serialisers/index.ts

```diff
@@ -0,0 +1,13 @@
+import user from './user';
+import visitor from './visitor';
+import volunteer from './volunteer';
+import volunteerLog from './volunteer_log';
+import communityBusiness from './community_business';
+
+export const Serialisers = {
```

Comment:
These method names are very general, each of the items could be serialised in multiple ways. I think it's worth being a bit more specific so we can serve future use cases without having to rename them.

Reply:
Agree in general, and that was my intent, but at the moment they're all being serialised in the same way.

Reply:
How about specifying in the name whether it is removing secrets or nothing?

```
Serialisers = {

  userSecret,
  visitorSecret,
  volunteerSecret,
  volunteerLogNone,
...
```

Reply:
yeah this is probs better. What about:
```
Serialisers.users.noSecrets
Serialisers.visitors.noSecrets
Serialisers.volunteers.noSecrets
Serialisers.volunteerLogs.identity
```

Reply:
Yeah, that works